src/Data/Array/NonEmpty/Internal.purs

```diff
@@ -1,4 +1,7 @@
-module Data.Array.NonEmpty.Internal (NonEmptyArray) where
+-- | This module exports the `NonEmptyArray` constructor, which can be
```

Comment:
I think it might be worth discouraging this module’s use a bit more, by adding a disclaimer that this module is not intended for public use and its API may change at any time, and a breaking change won’t be reflected in the version number?

Reply:
What do you think if this, with some bold all caps and newlines to let the disclaimer breath:

<img width="670" alt="Capture d’écran 2020-11-20 à 09 18 48" src="https://user-images.githubusercontent.com/2788549/99776730-8e7e1c80-2b11-11eb-8ec8-ca7bbd561143.png">

It could be nice of Pursuit to display a warning on internal modules, so we can have some CSS on top and we don’t have to synchronize them across packages.